Source/Engine/Platform/Win32/Win32Platform.cpp

```diff
@@ -212,9 +213,17 @@ bool Win32Platform::Init()
         DeviceId.D = (uint32)cpuInfo.ClockSpeed * cpuInfo.LogicalProcessorCount * cpuInfo.ProcessorCoreCount * cpuInfo.CacheLineSize;
     }
 
+    //TODO: log error if true
+    Win32Network::Init();
+    
     return false;
 }
 
+void Win32Platform::Exit()
+{
+    Network::Exit();
```

Comment:
Add `PlatformBase::Exit();` to call base (empty for now but who knows what's gonna be in future).